client/app/scripts/charts/edge.js

```diff
@@ -5,6 +5,30 @@ import classNames from 'classnames';
 import { enterEdge, leaveEdge } from '../actions/app-actions';
 import { encodeIdAttribute, decodeIdAttribute } from '../utils/dom-utils';
 
+function IsStorageComponent(id) {
```

Comment:
This should be `isStorageComponent `. function names are camelCase and start with a lowercase letter.